[2020-07-02 02:31:23,788] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: covid_data_dag.upload_data_mysql_task 2020-07-01T20:56:59.928337+00:00 [queued]>
[2020-07-02 02:31:23,800] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: covid_data_dag.upload_data_mysql_task 2020-07-01T20:56:59.928337+00:00 [queued]>
[2020-07-02 02:31:23,800] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-07-02 02:31:23,800] {taskinstance.py:880} INFO - Starting attempt 1 of 2
[2020-07-02 02:31:23,800] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-07-02 02:31:23,808] {taskinstance.py:900} INFO - Executing <Task(PythonOperator): upload_data_mysql_task> on 2020-07-01T20:56:59.928337+00:00
[2020-07-02 02:31:23,810] {standard_task_runner.py:53} INFO - Started process 91002 to run task
[2020-07-02 02:31:23,857] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: covid_data_dag.upload_data_mysql_task 2020-07-01T20:56:59.928337+00:00 [running]> localhost.localdomain
[2020-07-02 02:31:35,111] {logging_mixin.py:112} INFO - sqoop export   --table covid_data  --connect 'jdbc:mysql://localhost:3306/test'   --username root   --password 123Nineleap   --export-dir 'covid_data.csv'
[2020-07-02 02:31:35,112] {python_operator.py:114} INFO - Done. Returned value was: None
[2020-07-02 02:31:35,121] {taskinstance.py:1052} INFO - Marking task as SUCCESS.dag_id=covid_data_dag, task_id=upload_data_mysql_task, execution_date=20200701T205659, start_date=20200701T210123, end_date=20200701T210135
[2020-07-02 02:31:38,805] {logging_mixin.py:112} INFO - [2020-07-02 02:31:38,804] {local_task_job.py:103} INFO - Task exited with return code 0
